File: gitlfs-server/src/main/java/ru/bozaro/gitlfs/server/ForbiddenError.java
package ru.bozaro.gitlfs.server;

import jakarta.servlet.http.HttpServletResponse;

import javax.annotation.Nonnull;

/**
 * Forbidden error.
 *
 * @author Artem V. Navrotskiy
 */
public class ForbiddenError extends ServerError {
  public ForbiddenError() {
    this("Access forbidden");
  }

  public ForbiddenError(@Nonnull String message) {
    super(HttpServletResponse.SC_FORBIDDEN, message);
  }
}

File: gitlfs-server/src/main/kotlin/ru/bozaro/gitlfs/server/ForbiddenError.kt
package ru.bozaro.gitlfs.server

import jakarta.servlet.http.HttpServletResponse

/**
 * Forbidden error.
 *
 * @author Artem V. Navrotskiy
 */
class ForbiddenError constructor(message: String = "Access forbidden") :
        ServerError(HttpServletResponse.SC_FORBIDDEN, message)
